sqllogictest/src/parser.rs

```diff
@@ -632,72 +645,105 @@ mod tests {
         parse_roundtrip("../examples/basic/basic.slt")
     }
 
-    /// Parses the specified file into Records, and ensures the
-    /// results of unparsing them are the same
-    ///
-    /// Prints a hopefully useful message on failure
+    #[test]
+    fn test_condition() {
+        parse_roundtrip("../examples/condition/condition.slt")
+    }
+
+    #[test]
+    fn test_file_level_sort_mode() {
+        parse_roundtrip("../examples/file_level_sort_mode/file_level_sort_mode.slt")
+    }
+
+    #[test]
+    fn test_rowsort() {
+        parse_roundtrip("../examples/rowsort/rowsort.slt")
+    }
+
+    #[test]
+    fn test_test_dir_escape() {
+        parse_roundtrip("../examples/test_dir_escape/test_dir_escape.slt")
+    }
+
+    #[test]
+    fn test_validator() {
+        parse_roundtrip("../examples/validator/validator.slt")
+    }
+
+    /// Verifies Display impl is consistent with parsing by ensuring
```

Comment:
Removed the "display roundtrip check" (I will propose a more specific test in a follow on PR)